TWLight/applications/forms.py

```diff
@@ -319,3 +319,38 @@ def __init__(self, user=None, *args, **kwargs):
         # placeholders.
         self.fields['editor'].label = _('Username')
         self.fields['partner'].label = _('Partner name')
+
+
+class RenewalForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        try:
+            field_params = kwargs['field_params']
+        except KeyError:
+            logger.exception('Tried to instantiate a RenewalForm but '
+                'did not have field_params')
+            raise
+        self.field_params = kwargs.pop('field_params')
+        super(RenewalForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper(self)
+        self.helper.layout = Layout()
+        # add translator comment
+        fieldset = Fieldset(_('Renewal confirmation'))
+        if 'account_email' in self.field_params:
+            self.fields['account_email'] = forms.EmailField()
+            self.fields['account_email'].label = _('The email for your account on the partner\'s website')
```

Comment:
Bonus points if we can pre-fill this with the `account_email` data from the previous application!